demo/demo.js

```diff
@@ -142,18 +142,52 @@ document.getElementById("btcAddressSimple").addEventListener("click", async () =
 function makeExampleTx(keypathAccount) {
     const inputs = [
         {
-            "prevOutHash": new Uint8Array(32).fill(49), // arbitrary constant
+            "prevOutHash": Uint8Array.from([0xc5, 0x8b, 0x7e, 0x3f, 0x12, 0x00, 0xe0, 0xc0, 0xec, 0x9a, 0x5e, 0x81, 0xe9, 0x25, 0xba, 0xfa, 0xce, 0x2c, 0xc1, 0xd4, 0x71, 0x55, 0x14, 0xf2, 0xd8, 0x20, 0x5b, 0xe2, 0x50, 0x8b, 0x48, 0xee]),
```

Comment:
Is there an easy command go reconstruct this hash, in case I need to update the demo in the future?

Reply:
Would be nice to have a comment.

Reply:
There is on "easy" one that I know of. When I added the demo to the firmware repo, I just printed what the firmware computed for it to construct a demo that works.